Objects/Objects/BuiltElements/AdvanceSteel/AdvanceSteelObject.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Speckle.Core.Models;
+
+namespace Objects.BuiltElements.AdvanceSteel
+{
+  public class AdvanceSteelObject : Base
+  {
+    public AdvanceSteelObject()
+    {
+        
+    }
+
+    private string __typeAdvanceSteel;
+
+    public override string speckle_type
```

Comment:
Can you please explain this? I think we should not touch the `speckle_type` as it would affect deserialization

Reply:
Because of this call in ConnectorBindingsAutocad.
<img width="1059" alt="Screenshot 2023-03-13 at 14 43 21" src="https://user-images.githubusercontent.com/13578579/224784496-f2aa074e-1a9e-4c87-9124-5ba4caffb7ed.png">

I think changing just the AdvanceSteelObject won't affect the deserialization. The AdvanceSteelObject objects have specific names.
BTW, speckle_type is virtual in Base class. But if you want, we can create another property in Base Class to use in Report Object.